@here/olp-sdk-authentication/lib/UserAuth.ts

```diff
@@ -75,6 +75,10 @@ export interface UserAuthConfig {
      *
      */
     tokenRequester: TokenRequesterFn;
+    /**
+     * project scope
+     */
```

Comment:
This is not enough documentation. Please specify that in case of user specifies a project the resulting token is bound to this project and in case the desired resource is restricted to a project users need to specify this else they will not get a valid token.

Reply:
Updated